torch/fx/experimental/GraphManipulation.py

```diff
@@ -18,19 +20,19 @@ def find_use(arg: Any, node: Node) -> bool:
     else:
         return False
 
-def get_all_users_of(fx_module: GraphModule, index: int) -> List[int]:
-    """Given the graph(fx_module) and an index, return a list of all node indexes that use this node"""
+def get_all_users_of(fx_module: GraphModule, node: Node) -> List[Node]:
```

Comment:
Use node as reference instead of index, this function will be removed when the mutable graph API is landed.